lib/travis/build/addons/mariadb.rb

```diff
@@ -0,0 +1,33 @@
+require 'shellwords'
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Mariadb < Base
+        SUPER_USER_SAFE = true
+
+        MARIADB_GPG_KEY = '0xcbcb082a1bb943db'
+        MARIADB_MIRROR  = 'nyc2.mirrors.digitalocean.com'
```

Comment:
Not clear if this is the best mirror.

https://downloads.mariadb.org/mariadb/repositories/ shows James Madison University, which is physically closer to VA data centers.